File: Permutations with Largest Sum/permutationWithLargestSum.cpp
#include<bits/stdc++.h>
using namespace std;

// Function to calculate maximum sum for a permutation of array

void permutationWithLargestSum(int arr[], int N){
   
   sort(arr, arr + N); // sort the array
  
   for (int i = 0; i < N; i++)
	   cout<<arr[i]<<" ";

}

int main(){
    
    int N = 4;
    
    int arr[] = { 3, 5, 6, 1 };

    permutationWithLargestSum(arr, N);

    return 0;
    
}

File: Permutations with Largest Sum/permutationWithLargestSum.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Main{
	//This function finds permutation with largest sum.
	public static void permutationWithLargestSum(int[] arr){
		Arrays.sort(arr);
		for (int i=0;i<arr.length;i++) System.out.print(arr[i] + " ");
	}
	//Main Function
	public static void main(String[] args){
		int n = 4;
		int[] arr = {3,5,6,1};
        permutationWithLargestSum(arr);
	}
}